pycode/memilio-epidata/memilio/epidata/geoModificationGermany.py

```diff
@@ -491,6 +491,8 @@ def merge_df_counties(
     @return Reduced data frame with separated_ids information merged to 
         merged_id rows.
     """
+    # verify that IDs are integers
+    df[dd.EngEng['idCounty']] = df[dd.EngEng['idCounty']].astype(int)
```

Comment:
This is true and an important issue that I tried to resolved to easily. Thank you. I hope to provide a better solution now. The idea is that 

`df[dd.EngEng['idCounty']].isin(separated_ids)`

works. Otherwise there is no need for a particular datatype. Unfortunately, dataframe integers are sometimes read in as objects/strings so that the previous method fails without an error (i.e., it does nothing).